src/v2/models.py

```diff
@@ -147,37 +144,21 @@ class PackageDetails(Package):  # noqa: D101
     url: Optional[str] = None
 
 
-class PackageDetailsForRegisteredUser(PackageDetails):  # noqa: D101
-    public_vulnerabilities: Optional[List['PremiumVulnerabilityFields']] = Field(
-        None, description='Publicly known vulnerability details'
+class StackAggregatorPackageData(PackageDetails):  # noqa: D101
```

Comment:
This is not possible, as 'PackageDetails' is used for building 'RecommendedPackageData' also.

Reply:
I don't this going to be an issue because from Recommender public_vulnerabilities, private_vulnerabilities and recommended_version can be set to empty.

Reply:
If we do that, our API response will not be as per API spec definition, also having non-related fields in 'recommendation' section is not good. Also we don't gain too much by eliminating one data model. Leaving code as is.

Reply:
Okay. Can you rename it to `PackageDataWithVulnerabilities` and update the spec? It looks appropriate than generic `StackAggregatorPackageData`.